fix sharding-spring-boot-mybatis-example

## Bug Report


### Which version of ShardingSphere did you use?

5.0.0-RC1-SNAPSHOT

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?

ShardingSphere-JDBC



### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.

 1.  update  application.properties  
   `spring.profiles.active=master-slave`
 2. Run  org.apache.shardingsphere.example.sharding.spring.boot.mybatis.ExampleMain


**Exception message**:

`[INFO ] 2020-09-02 11:34:13,726 --main-- [org.apache.shardingsphere.example.sharding.spring.boot.mybatis.ExampleMain] The following profiles are active: master-slave 
[INFO ] 2020-09-02 11:34:13,981 --main-- [org.springframework.context.annotation.AnnotationConfigApplicationContext] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@3bd40a57: startup date [Wed Sep 02 11:34:13 CST 2020]; root of context hierarchy 
[ERROR] 2020-09-02 11:34:14,469 --main-- [org.springframework.boot.SpringApplication] Application startup failed 
org.apache.shardingsphere.infra.spi.exception.ServiceProviderNotFoundException: No implementation class load from SPI `org.apache.shardingsphere.masterslave.spi.MasterSlaveLoadBalanceAlgorithm` with type `ROUND_ROB`.
	at org.apache.shardingsphere.infra.spi.type.TypedSPIRegistry.getRegisteredService(TypedSPIRegistry.java:50)
	at org.apache.shardingsphere.spring.boot.registry.AbstractAlgorithmProvidedBeanRegistry.lambda$registerBean$2(AbstractAlgorithmProvidedBeanRegistry.java:63)
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:684)
	at org.apache.shardingsphere.spring.boot.registry.AbstractAlgorithmProvidedBeanRegistry.registerBean(AbstractAlgorithmProvidedBeanRegistry.java:62)
	at org.apache.shardingsphere.masterslave.spring.boot.algorithm.MasterSlaveAlgorithmProvidedBeanRegistry.postProcessBeanDefinitionRegistry(MasterSlaveAlgorithmProvidedBeanRegistry.java:44)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanDefinitionRegistryPostProcessors(PostProcessorRegistrationDelegate.java:272)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(PostProcessorRegistrationDelegate.java:122)
	at org.springframework.context.support.AbstractApplicationContext.invokeBeanFactoryPostProcessors(AbstractApplicationContext.java:687)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:525)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:737)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:370)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:314)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1162)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1151)
	at org.apache.shardingsphere.example.sharding.spring.boot.mybatis.ExampleMain.main(ExampleMain.java:37)
[INFO ] 2020-09-02 11:34:14,471 --main-- [org.springframework.context.annotation.AnnotationConfigApplicationContext] Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@3bd40a57: startup date [Wed Sep 02 11:34:13 CST 2020]; root of context hierarchy`

--- a/examples/shardingsphere-jdbc-example/sharding-example/sharding-spring-boot-mybatis-example/src/main/resources/application-master-slave.properties
+++ b/examples/shardingsphere-jdbc-example/sharding-example/sharding-spring-boot-mybatis-example/src/main/resources/application-master-slave.properties
@@ -28,7 +28,7 @@ spring.shardingsphere.datasource.ds_slave_0.jdbc-url=jdbc:mysql://localhost:3306
 
 spring.shardingsphere.datasource.ds_slave_1.jdbc-url=jdbc:mysql://localhost:3306/demo_ds_slave_1?serverTimezone=UTC&useSSL=false&useUnicode=true&characterEncoding=UTF-8
 
-spring.shardingsphere.rules.master-slave.load-balancers.round_robin.type=ROUND_ROB
+spring.shardingsphere.rules.master-slave.load-balancers.round_robin.type=ROUND_ROBIN
 spring.shardingsphere.rules.master-slave.data-sources.ds_ms.master-data-source-name=ds_master
 spring.shardingsphere.rules.master-slave.data-sources.ds_ms.slave-data-source-names=ds_slave_0,ds_slave_1
 spring.shardingsphere.rules.master-slave.data-sources.ds_ms.load-balancer-name=round_robin

